rhel/rhcc/rhcc.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/quay/claircore/toolkit/types/cpe"
 )
 
-var goldRepo = claircore.Repository{
+var GoldRepo = claircore.Repository{
```

Comment:
nit: Maybe add a godoc for this since it's now available outside the package

Reply:
Will do this in https://github.com/quay/claircore/pull/1416 as it will (hopefully) be merged first.